### PR TITLE
ref: Refactor space validation and use Tune components

### PR DESCRIPTION
**Pull request by samuveth**
### Issues

Fixes https://github.com/snapshot-labs/snapshot/issues/3797

### Changes 

1. Use Tune component for Settings and Setup components
2. Settings Save button is now always clickable and will show all error messages on click
3. Show error message for all fields at the top of Settings Screenshot 2023-04-23 at 5 33 11 PM

### Self-review checklist
- [x] I have performed a full self-review of my changes
- [x] I have tested my changes on a preview deployment
- [x] I have tested my changes on a custom domain
- [ ] I have run end-to-end tests `yarn cypress:test:e2e`, and they have passed

### CODE REVIEW

**Comment by samuveth** [comment]
Tested on multiple custom skins and made some fixes

File: src/components/BaseModalSelectItem.vue [hunk 1]
```diff
@@ -15,7 +15,7 @@ defineProps<{
     ]"
   >
     <div class="relative inset-y-0 flex items-center">
-      <div :class="['w-full', { 'pr-[44px]': selected }]">
+      <div :class="['w-full text-left', { 'pr-[44px]': selected }]">
```
Comment (ChaituVR) [inline src/components/BaseModalSelectItem.vue:18]: Any reason for text-left here? it is left without it also no? 🤔

File: src/components/SettingsMembersBlock.vue [hunk 2]
```diff
@@ -240,12 +240,12 @@ const errorMessage = computed(() => {
 
     <div class="mt-3">
       <div class="flex items-end gap-1">
-        <BaseInput
+        <TuneInput
```
Comment (ChaituVR) [inline src/components/SettingsMembersBlock.vue:243]: I don't like the design for error messages 🙈 previous one was nice :D 
Also, the alignment here looks weird
Untitled

Previous: 
image

File: src/components/BaseModalSelectItem.vue (hunk 1 above)
Comment (samuveth) [reply src/components/BaseModalSelectItem.vue:18]: Nope, this was a bug I discovered and added to this PR. Will hotfix in on prod now instead.
image

File: src/style.scss
Comment (ChaituVR) [inline src/style.scss:1]: These "Tune component styles" are like overrides that are only for snapshot? if not these should be in tune library no? 🤔

Comment (samuveth) [reply src/style.scss:1]: They are only for snapshot correct.

Comment (samuveth) [reply src/style.scss:1]: I see now that some could be moved to Tune actually :)

File: src/components/SettingsMembersBlock.vue (hunk 2 above)
Comment (ChaituVR) [reply src/components/SettingsMembersBlock.vue:243]: Cool cool :) can move the Author selection menu bit up?? 🙈 the alignment looks weird with the error message 😅

Comment (samuveth) [reply src/components/SettingsMembersBlock.vue:243]: Yep will fix that of course :)

**Pull request merged**